search/mongo_search.go

```diff
@@ -14,6 +14,8 @@ import (
 	"gopkg.in/mgo.v2/bson"
 )
 
+var opInterruptedCode = 11601
```

Comment:
When specifying "magic numbers" it's always nice to add some documentation.  For example, perhaps a pointer to where the magic number is actually defined/specified by Mongo.  (If possible).